File: src/gameEntities/modifiers/HealthUpgrade1.java
package gameEntities.modifiers;


import dynamicBody.character.player.Stats;
import worldModel.utilities.Pair;
import worldModel.utilities.enums.Entities;

public class HealthUpgrade1 extends ModifiersImpl{

	public HealthUpgrade1(Pair<Integer, Integer> coord) {
		super(coord, Stats.HEALTH, 10, "Health Upgrade", Entities.HEALTHUPGRADE1);
	}
}

File: New folder/GameEntity/HealthUpgrade1.cs
using System;

using design.utilities;
using design.enums;

namespace gameEntities.modifiers
{
    public class HealthUpgrade1 : ModifiersImpl
    {
        public HealthUpgrade1(Pair<int, int> coord) 
            : base(coord, Stats.HEALTH, 10, "Health Upgrade", Entities.HEALTHUPGRADE1)
        {

        }

    }
}
